inst/www/shared/shiny.js

```diff
@@ -3477,7 +3466,6 @@ $(document).on('click', '.slider-animate-button', function(evt) {
           val.to = Math.min(slider.result.max, slider.result.to + slider.options.step);
 
         slider.update(val);
-        forceIonSliderUpdate(slider);
```

Comment:
They were from an earlier commit to master that didn't get grunt-built properly. (Ideally we would have travis detect that condition)